### PR TITLE
Bug 1759667: Add the official 4.2 channels to cluster upgrade settings

### PR DESCRIPTION
**Pull request by spadgett**
/assign @rhamilto @jwforres 
/hold

### CODE REVIEW

File: frontend/public/module/k8s/cluster-settings.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ export const getAvailableClusterChannels = () => ({
   'prerelease-4.1': 'prerelease-4.1',
   'stable-4.1': 'stable-4.1',
   'prerelease-4.2': 'prerelease-4.2',
+  'stable-4.2': 'stable-4.2',
```
Comment (smarterclayton) [inline frontend/public/module/k8s/cluster-settings.ts]: We're still debating the next name to add.  It will be one other name.

Comment (smarterclayton) [reply frontend/public/module/k8s/cluster-settings.ts]: Add `fast-4.2` and `candidate-4.2`, remove `prerelease-4.2`.

**Comment by spadgett** [comment]
@smarterclayton updated

Cluster Settings · OKD 2019-10-10 12-59-22

File: frontend/public/module/k8s/cluster-settings.ts (hunk 1 above)
Comment (smarterclayton) [reply frontend/public/module/k8s/cluster-settings.ts]: Ordering wise, will stable show up at top, then fast, then candidate?

**Comment by smarterclayton** [comment]
See my order comment

**Comment by spadgett** [comment]
I updated the ordering:

Cluster Settings · OKD 2019-10-10 13-16-09

**Pull request merged**